Source/PLCrashLogWriter.m

```diff
@@ -1358,6 +1384,18 @@ plcrash_error_t plcrash_log_writer_write (plcrash_log_writer_t *writer,
         plcrash_writer_write_binary_image(file, &image->macho_image);
     }
 
+    /* Try to read crash reporter annotations */
+    struct crashreporter_annotations_t *annotations = NULL;
+    image = plcrash_async_image_containing_address(image_list, siginfo->bsd_info->address);
+    if (image != NULL)
+    {
+        plcrash_async_mobject_t mobj;
+        err = plcrash_async_macho_map_section(&image->macho_image, "__DATA", CRASHREPORTER_ANNOTATIONS_SECTION, &mobj);
+        if (err == PLCRASH_ESUCCESS) {
```

Comment:
Nitpick: what is the formatting style here wrt to brackets? Choose one.

Reply:
like in this line. I've already noticed about it myself